_includes/navigation.html

```diff
@@ -27,6 +27,6 @@
         </ul>
     </div>
     <div class="ctrl-right">
-        <a href="javascript:void(0)" id="menu-toggle" aria-label="Current page's menu toggle"><i class="fa fa-indent" aria-hidden="true"></i></a>
+        <a href="javascript:void(0)" id="menu-toggle" aria-label="Current page's menu toggle"></a>
```

Comment:
Looks like this tag is actually used to show the icon, so I think we should remove this commit 😅

Reply:
That kind of makes sense! I never knew an <i> tag would be used to build a menu, they said the <i> tag was used to italicize text 😃. My whole life was a lie 🤯.